.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,2 +1,13 @@
+arguments=
```

Comment:
The added settings all seem fairly generic; are these really required? (I'm too out-of-touch with Eclipse to know which parts of the project metadata should be committed to VCS and which don't.)

Reply:
I did not configure anything manually, but eclipse keeps automatically changing this file. It's annoying to always deal with it as unstaged changes, so I just committed it. If it's not needed we can add it to the .gitignore, but I don't know enough about the project files to make that decision.

Reply:
I have the feeling that I'll learn about the project settings when looking into #10, so I'm going to defer a decision until then.